websocket.go

```diff
@@ -357,10 +357,14 @@ func (ce *cerror) Error() string {
 }
 
 // getWebHost returns the host:port+1 of the serverIdentity.
-func getWebHost(si *network.ServerIdentity) (string, error) {
+func getWebHost(si *network.ServerIdentity, server bool) (string, error) {
```

Comment:
+ `getWebAddress` seems more appropriate now, 
+ `server` => `global` 
+ A small comment saying what is this boolean doing.